### ISSUE TITLE
Ipxe does not load - operation not permitted.

### ISSUE THREAD

**Issue by Kamilcuk** [alpinelinux/alpine-netboot#2]
I downloaded image from https://boot.alpinelinux.org/

I'm getting: `Operation not permitted (http://ipxe.org/410de13c)` , both on PC and in qemu.

![image](https://user-images.githubusercontent.com/15679217/146620887-47c0498e-7df3-4b9f-afb2-b4a87721eff9.png)

**Comment by Wyk72** [comment]
I have same the problem, both PC/UEFI & QEMU/lkrnl. It seems this project is dead.

**Comment by pide2000** [comment]
Its about the cipher suites. You try to load from https, most servers don't support the (old) cipher suites of ipxe because of security issues. 
For a full list see: https://ipxe.org/crypto

**Comment by clandmeter** [comment]
@pide2000 you are correct, this is what i debugged last month but didnt find time to work out a solution.
The problem is that most of our front facing http proxies all have sane defaults, so i need to see if i can fix it differently.

**Comment by pide2000** [comment]
~~Using this mirror works for us.~~
~~http://dl-cdn.alpinelinux.org/alpine~~
~~Because of imgverify it shouldn't be a problem to use plain http.~~
~~Perfect solution would be to implement modern cipher suites in ipxe but I think this wouldn't happen in the near future.~~

Sry, this is simply not true. The mirror above redirects to https, we use these mirrors in a later stage of the boot process where https ist provided by the OS. For now we provide the kernel and initrd from a local http server. 
For testing purposes one might download the netboot version of alpine and start a http server in the folder where alpine kernel and initrd is located with `python -m http.server 80`
